chapter_convolutional-neural-networks/googlenet.md

```diff
@@ -1,6 +1,6 @@
 # Networks with Parallel Concatenations (GoogLeNet)
 
-During the ImageNet Challenge in 2014, a new architeture emerged that outperformed the rest. [Szegedy et al., 2014](https://arxiv.org/abs/1409.4842) proposed a structure that combined the strenghts of the NiN and repeated blocks paradigms. At its heart was the rather pragmatic answer to the question as to which size of convolution is ideal for processing. After all, we have a smorgasbord of choices, $1 \times 1$ or $3 \times 3$, $5 \times 5$ or even larger. And it isn't always clear which one is the best. As it turns out, the answer is that a combination of all the above works best. Over the next few years, researchers made several improvements to GoogLeNet. In this section, we will introduce the first version of this model series in a slightly simplified form - we omit the pecularities that were added to stabilize training, due to the availability of better training algorithms.
+During the ImageNet Challenge in 2014, a new architecture emerged that outperformed the rest. [Szegedy et al., 2014](https://arxiv.org/abs/1409.4842) proposed a structure that combined the strengths of the NiN and repeated blocks paradigms. At its heart was the rather pragmatic answer to the question as to which size of convolution is ideal for processing. After all, we have a smorgasbord of choices, $1 \times 1$ or $3 \times 3$, $5 \times 5$ or even larger. And it isn't always clear which one is the best. As it turns out, the answer is that a combination of all the above works best. Over the next few years, researchers made several improvements to GoogLeNet. In this section, we will introduce the first version of this model series in a slightly simplified form - we omit the peculiarities that were added to stabilize training, due to the availability of better training algorithms. Notable, the original architecture included several "auxiliary classification" heads which addressed the diminution of gradients in very deep networks; this slightly cumbersome technique has become less popular over time [3].
 
```

Comment:
Thanks. It would still be hard for readers without much background to understand the descriptions here: auxiliary classification, diminution of gradients, etc.. Could you revert such changes?